### PR TITLE
21-0972 content updates

### PR DESCRIPTION
**Pull request by Thrillberg**
## Summary

This PR adds mostly typographical content updates to form 21-0972. In addition to the typographical changes, this also adds 4 page variants for the preparer qualifications page, depending on the claimant's relationship to the veteran.

## Related issue(s)
https://app.zenhub.com/workspaces/vagov-product-team-forms-634853151f5c6000165942bc/issues/gh/department-of-veterans-affairs/va.gov-team-forms/442

## Testing done

Tests were already in place but I added tests for the page that now has 4 variants.

## Screenshots

Screenshot 2023-08-07 at 2 38 13 PM

If claimant is the Veteran.

Screenshot 2023-08-07 at 2 38 58 PM

If claimant is Veteran's parent.

## What areas of the site does it impact?

form 21-0972

### Quality Assurance & Testing

- [X] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [X] Linting warnings have been addressed
- [X] Screenshot of the developed feature is added

### Error Handling

- [X] Browser console contains no warnings or errors.
- [X] Events are being sent to the appropriate logging solution

### CODE REVIEW

File: src/applications/simple-forms/21-0972/definitions/constants.js [hunk 1]
```diff
@@ -6,10 +6,10 @@ export const claimantIdentificationKeys = [
 ];
 
 export const claimantIdentificationOptions = {
```
Comment (tlei123) [inline src/applications/simple-forms/21-0972/definitions/constants.js:8]: Better to use Object.freeze({...}) to truly prevent the _props_ from being inadvertantly overwritten in your app-code.  [i'm circling back to my older forms to do the same.]

File: src/applications/simple-forms/21-0972/config/form.js [hunk 2]
```diff
@@ -45,46 +46,50 @@ const statementOfTruthBody = (
...
   </>
 );
 
 const formConfig = {
   rootUrl: manifest.rootUrl,
   urlPrefix: '/',
-  submitUrl: `${environment.API_URL}/simple_forms_api/v1/simple_forms`,
+  // submitUrl: `${environment.API_URL}/simple_forms_api/v1/simple_forms`,
+  submit: () =>
+    Promise.resolve({ attributes: { confirmationNumber: '123123123' } }),
```
Comment (rhasselle-oddball) [inline src/applications/simple-forms/21-0972/config/form.js:88]: do we want to commit this? or was this just for testing?

**Review by rhasselle-oddball — APPROVED** [review]
lots of content changes! looks good as far as I can tell.

File: src/applications/simple-forms/21-0972/config/form.js (hunk 2 above)
Comment (Thrillberg) [reply src/applications/simple-forms/21-0972/config/form.js:88]: Whoopsie! Good call. Thanks!

File: src/applications/simple-forms/21-0972/pages/preparerQualifications1.js [hunk 3]
```diff
@@ -1,22 +1,88 @@
...
+    'ui:options': {
+      forceDivWrapper: true,
+      // different labels between all the variants
+      showFieldLabel: true,
+    },
+  },
+};
 export default {
-  uiSchema: {
+  uiSchemaA: {
```
Comment (rhasselle-oddball) [inline src/applications/simple-forms/21-0972/pages/preparerQualifications1.js:25]: can you keep all of the uiSchema and schema without a postfix? the intellisense that /** @type {PageSchema} */ helps with (when/if loaded) expects uiSchema and schema explicitly in order to work.

Comment (rhasselle-oddball) [reply src/applications/simple-forms/21-0972/pages/preparerQualifications1.js:25]: let's sync on this topic offline

**Pull request merged**